frontend/src/components/feed/Feed.js

```diff
@@ -30,8 +34,10 @@ const Feed = ({ navigate }) => {
     if(token) {
       return(
         <>
-          <h2>Posts</h2>
-            <button onClick={logout}>
+          <h2 className='h2'>Posts</h2>
+            <button className='btn' onClick={() => setForm(!form) }>Add Post</button>
```

Comment:
Can I check if these buttons are/could be components? Might want to rename the class so that we know what that button is doing?

Reply:
i agree